### PR TITLE
feat: Add additional metadata for PowerShell Gallery listing

### PR DESCRIPTION
**Pull request by tenthirtyam**
In order to have a good experience with our community, we recommend that you read the [contributing guidelines](https://github.com/vmware-samples/power-validated-solutions-for-cloud-foundation/blob/main/CONTRIBUTING.md) for making a pull request.

**Summary of Pull Request**

- Adds additional metadata to PowerShell Gallery listing:
  - Tags
  - Project URL
  - License URL
  - Release Notes URL (Changelog)
  - Icon URL

- Adds the .svg and .png files for the `IconUri` module's PSData setting and the `README.md`.

- Updates the `README.me` with additional context for description, requirements, links,  help, and formatting.

> **Note**
>
> Once there is a new release to the PSGallery, the `PSData` will add the metadata to the listing.

Signed-off-by: Ryan Johnson <johnsonryan@vmware.com>

**Type of Pull Request**

- [ ] This is a bug fix.
- [x] This is an enhancement or feature.
- [ ] This is a code style / formatting update.
- [x] This is a documentation update.
- [ ] This is a refactoring update.
- [ ] This is something else.
      Please describe:

**Related to Existing Issues**

Resolves: #74 

**Test and Documentation Coverage**

- [ ] Tests have been completed (for bug fixes / features).
- [x] Documentation has been added / updated (for bug fixes / features).

**Breaking Changes?**

- [ ] Yes, there are breaking changes.
- [x] No, there are no breaking changes.

### CODE REVIEW

**Review by GaryJBlake — COMMENTED** [review]
Few minors fixes needed

File: README.md [hunk 1]
```diff
@@ -1,20 +1,45 @@
...
 ![](https://img.shields.io/powershellgallery/dt/PowerValidatedSolutions?style=for-the-badge)
 
-# PowerValidatedSolutions
 ## Overview
-PowerValidatedSolutions is a PowerShell Module that has been written to explicitly support the automation of various procedures associated with the implementation of Validated Solutions for VMware Cloud Foundation. 
 
-The intent is to reduce human error, deliver consistency and robustness as well as expedite the time to deploy for customers by developing cmdlets that remove the complexities associated with performing deployment and configuration steps using Public/Private APIs or command line based tooling.
+`PowerValidatedSolutions` is a PowerShell module that has been written to support the automation of many procedures associated with the implementation of the [VMware Validated Solutions for VMware Cloud Foundation][vvs].
+
+The module helps reduce human error, delivers consistency and robustness, and accelerates the time-to-deploy these solutions. The module's cmdlets reduce complexity and by automating the deployment and configuration steps using the product APIs or command-line tools.
```
[the remark this answers is not in the document]
Comment (tenthirtyam) [reply README.md]: ✅ Addressed in commit https://github.com/vmware-samples/power-validated-solutions-for-cloud-foundation/pull/76/commits/82b214ff4975ce259f3c08d52619997d8bd660c4.

File: README.md [hunk 2]
```diff
@@ -24,22 +49,43 @@ Install-Module -Name PowerValidatedSolutions -MinimumVersion 1.7.0
...
 
-```PowerShell
+```powershell
 Get-InstalledModule
 ```
 
-Once installed, any new cmdlet associated with PowerValidatedSolutions will be available for use.
+Once installed, any cmdlets associated with `PowerValidatedSolutions` and the supporting PowerShell modules will be available for use.
+
+To view the cmdlets for available in the module, run the following command in the PowerShell console.
```
[the remark this answers is not in the document]
Comment (tenthirtyam) [reply README.md]: ✅ Addressed in commit https://github.com/vmware-samples/power-validated-solutions-for-cloud-foundation/pull/76/commits/82b214ff4975ce259f3c08d52619997d8bd660c4.

File: README.md [hunk 3]
```diff
@@ -1,20 +1,45 @@
...
-# PowerValidatedSolutions
 ## Overview
-PowerValidatedSolutions is a PowerShell Module that has been written to explicitly support the automation of various procedures associated with the implementation of Validated Solutions for VMware Cloud Foundation. 
 
-The intent is to reduce human error, deliver consistency and robustness as well as expedite the time to deploy for customers by developing cmdlets that remove the complexities associated with performing deployment and configuration steps using Public/Private APIs or command line based tooling.
+`PowerValidatedSolutions` is a PowerShell module that has been written to support the automation of many procedures associated with the implementation of the [VMware Validated Solutions for VMware Cloud Foundation][vvs].
+
+The module helps reduce human error, delivers consistency and robustness, and accelerates the time-to-deploy these solutions. The module's cmdlets reduce complexity and by automating the deployment and configuration steps using the product APIs or command-line tools.
+
+This infrastructure-as-code model enables you to use the module "out-of-the-box" or develop incorporate into your own end-to-end automation.
```
[the remark this answers is not in the document]
Comment (tenthirtyam) [reply README.md]: ✅ Addressed in commit https://github.com/vmware-samples/power-validated-solutions-for-cloud-foundation/pull/76/commits/82b214ff4975ce259f3c08d52619997d8bd660c4.

**Comment by tenthirtyam** [comment]
✅ All review comments addressed in commit https://github.com/vmware-samples/power-validated-solutions-for-cloud-foundation/pull/76/commits/82b214ff4975ce259f3c08d52619997d8bd660c4.

**Comment by GaryJBlake** [comment]
LGTM

**Pull request merged**